src/config/toml/config.rs

```diff
@@ -69,3 +72,158 @@ pub(crate) struct GitHub {
     /// The name of the repository in GitHub that this project is utilizing
     pub(crate) repo: String,
 }
+
+/// Details needed to use steps that interact with a Gitea instance.
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[cfg_attr(test, derive(PartialEq, Eq))]
+pub(crate) struct Gitea {
+    /// The user or organization that owns the `repo`.
+    pub(crate) owner: String,
+    /// The name of the repository
+    pub(crate) repo: String,
+    /// The domain or IP of the Gitea instance
```

Comment:
```suggestion
    /// The domain of the Gitea instance
```

We're assuming https later, so I don't think an IP would work. Maybe that's also worth calling out in the docs for config.

Reply:
HTTPS is only assumed for known public instances ^^